### PR TITLE
Add blog post strip to /observability

### PR DESCRIPTION
**Pull request by mmanciop**
## Done

- Add blog post strip for the `observability` tag

## QA

- Check out this feature branch
- Run the site using the command `./run serve` or `dotrun`
- View the site locally in your web browser at: http://0.0.0.0:8001/
    - Be sure to test on mobile, tablet and desktop screen sizes
- [List additional steps to QA the new features or prove the bug has been resolved]

### CODE REVIEW

**Comment by mmanciop** [comment]
mmm I am fairly sure the tag is the right one, but in demohaus I see FinOps posts?

**Review by anthonydillon — CHANGES_REQUESTED** [review]
For consistency, could you move the blog section above the `contact-form-container` please

**Comment by mmanciop** [comment]
> For consistency, could you move the blog section above the `contact-form-container` please

Done

**Comment by mmanciop** [comment]
@anthonydillon I am somewhat worried that demohaus showing the Finance blog posts instead of the Observability ones is not just a fluke of the demo setup. How can this be validated?

**Comment by anthonydillon** [comment]
Ah ok, checked and the tag id for observability is 4125. If you switch that in the code it should pull the right content in.

**Comment by mmanciop** [comment]
Yep, looks good @anthonydillon . Since there was already the `tag_name`, I completely missed out on `tag_id`.

**Comment by mmanciop** [comment]
mmm the rendering puts the blogs stripe _under_ the contact form below the content form despite having it move it above it. What is this sorcery.

**Comment by mmanciop** [comment]
> mmm the rendering puts the blogs stripe _under_ the contact form below the content form despite having it move it above it. What is this sorcery.

Yeah my bad. I mistook the `div` for the contact form holder for the contact form itself.

File: templates/observability/index.html [hunk 1]
```diff
@@ -634,7 +634,11 @@ <h3>Open source, simplified</h3>
   <hr />
 </div>
 
-<section class="p-strip is-deep">
+{% with section_classes='p-strip', heading_topic='Observability', tag_name='observability', tag_id='4125', limit='4' %}
+  {% include "shared/_latest_news_strip.html" %}
+{% endwith %}
+
+<section class="p-strip">
```
Comment (anthonydillon) [inline templates/observability/index.html:641]: I'm no designer but this might fit better if it was `p-strip--light`

**Review by anthonydillon — APPROVED** [review]
Code and QA LGTM :+1: just need a design review

File: templates/observability/index.html (hunk 1 above)
Comment (mmanciop) [reply templates/observability/index.html:641]: Yeah, I tried it locally and I like it better :-)

File: templates/observability/index.html [hunk 2]
```diff
@@ -355,12 +355,15 @@ <h3 class="p-heading--four">Dashboards and Alerting</h3>
   </div>
 </section>
 
-<section class="p-strip is-deep">
-  <div class="row">
-    <div class="u-fixed-width">
-      <h2 class="u-sv3">Latest Ubuntu Observability news from <a href="/blog/tag/observability">our blog</a>&nbsp;&rsaquo;</h2>
-    </div>
-  </div>
+<div class="u-fixed-width">
```
Comment (pmahnke) [inline templates/observability/index.html:358]: I think this hr and div isn't required as the above section is-bordered

Comment (mmanciop) [reply templates/observability/index.html]: Done

**Pull request merged**